include/wx/textctrl.h

```diff
@@ -270,6 +270,18 @@ enum wxTextAttrLineSpacing
     wxTEXT_ATTR_LINE_SPACING_TWICE          = 20
 };
 
+#if wxUSE_SPELLCHECK
+/*!
```

Comment:
Ah, okay - I was following the style for the two previous enums (wxTextAttrEffects and wxTextAttrLineSpacing). I've removed the whole comment because it is a bit superfluous. Also, I've update the name to wxTextSpellCheckStatus (generally changing _Spellcheck_ to _SpellCheck_)

Reply:
Noted. I have removed this whole change as we don't need the enum anymore.